tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
     "resolveJsonModule": true,
-    "preserveWatchOutput": true
+    "preserveWatchOutput": true,
+    "types": ["node", "jest", "@testing-library/jest-dom"]
```

Comment:
@testing-library/jest-dom adds more matchers to jest's `expect(...)`; for example, `expect(element).toBeInTheDocument()`. In v6, the mechanism that informs typescript about the new matchers changed.

See https://github.com/testing-library/jest-dom/issues/517 for a little more info.